setup-homebrew/main.sh

```diff
@@ -231,20 +242,24 @@ else
     done
 fi
 
-if [[ "${TEST_BOT}" == 'true' ]]; then
+if [[ "${TEST_BOT}" == "true" ]] || [[ "${TEST_BOT}" == "auto" && -n "${HOMEBREW_TAP_REPOSITORY-}" ]]; then
     # Setup Homebrew/homebrew-test-bot
     HOMEBREW_TEST_BOT_REPOSITORY="$HOMEBREW_REPOSITORY/Library/Taps/homebrew/homebrew-test-bot"
-    if ! [[ -d "$HOMEBREW_TEST_BOT_REPOSITORY" ]]; then
-        ohai "Fetching Homebrew/test-bot..."
-        git_retry clone https://github.com/Homebrew/homebrew-test-bot "$HOMEBREW_TEST_BOT_REPOSITORY"
-    elif [[ "$GITHUB_REPOSITORY" != "Homebrew/homebrew-test-bot" ]]; then
+    if [[ -d "$HOMEBREW_TEST_BOT_REPOSITORY" ]]; then
         ohai "Fetching Homebrew/test-bot..."
         git_retry -C "$HOMEBREW_TEST_BOT_REPOSITORY" fetch --force origin
         git -C "$HOMEBREW_TEST_BOT_REPOSITORY" remote set-head origin --auto
         git -C "$HOMEBREW_TEST_BOT_REPOSITORY" checkout --force -B master origin/HEAD
+    else
+        [[ "$GITHUB_REPOSITORY" != "Homebrew/homebrew-test-bot" ]]
```

Comment:
There's a floating `[[ "$GITHUB_REPOSITORY" != "Homebrew/homebrew-test-bot" ]]` here

```suggestion
    if [[ -d "$HOMEBREW_TEST_BOT_REPOSITORY" && "$GITHUB_REPOSITORY" != "Homebrew/homebrew-test-bot" ]]; then
        ohai "Fetching Homebrew/test-bot..."
        git_retry -C "$HOMEBREW_TEST_BOT_REPOSITORY" fetch --force origin
        git -C "$HOMEBREW_TEST_BOT_REPOSITORY" remote set-head origin --auto
        git -C "$HOMEBREW_TEST_BOT_REPOSITORY" checkout --force -B master origin/HEAD
    else
```